experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -855,17 +858,38 @@ describe("FormAudience", () => {
     expect(screen.queryByTestId("locales")).not.toBeInTheDocument();
   });
 
-  it("isFirstRun renders for mobile application", async () => {
-    render(
-      <Subject
-        experiment={{
-          ...MOCK_EXPERIMENT,
-          application: NimbusExperimentApplicationEnum.FENIX,
-        }}
-      />,
-    );
+  Object.values(NimbusExperimentApplicationEnum).forEach((application) => {
+    if (MOBILE_APPLICATIONS.includes(application)) {
+      it(`isFirstRun renders for mobile application ${application}`, async () => {
+        render(
+          <Subject
+            experiment={{
+              ...MOCK_EXPERIMENT,
+              application,
+            }}
+          />,
+        );
+
+        expect(screen.queryByTestId("isFirstRun")).toBeInTheDocument();
+      });
+    }
+  });
 
-    expect(screen.queryByTestId("isFirstRun")).toBeInTheDocument();
+  Object.values(NimbusExperimentApplicationEnum).forEach((application) => {
+    if (!MOBILE_APPLICATIONS.includes(application)) {
+      it(`isFirstRun does not render for non mobile application ${application}`, async () => {
```

Comment:
Prefer it.each()?


```js
it.each(
  Object.values(NimbusExperimentApplicationEnum).filter(application => !MOBILE_APPLICATIONS.includes(application)
)("msg", (application) => {
/* test */
});
```

see validators.test.ts:164 for an example

Reply:
Thank you! I knew there had to be a better.